packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -15,6 +15,8 @@ Each org starts off with Okta's default branding. You can upload your own assets
 
 Explore the Brands API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/8cc47beb2a20dfe078eb)
 
+> **Note:** This Postman collection requires the <ApiLifecycle access="ea"/> Loading Page Animation feature to be enabled.
```

Comment:
should we be more specific about why this will be required? something like:
```
> **Note:** This Postman collection requires the <ApiLifecycle access="ea"/> Loading Page Animation feature to be enabled if using Themes API with `loadingPageTouchPointVariant`.
```